### DISCUSSION TITLE
PCB Adapters not on six-slot adapter

### DISCUSSION THREAD

**Discussion by ccfman2004** [sanni/cartreader#560] [Need help]
Is there a list of where I can find the gerber files for the slot adapters not on the six-slot adapter?

**Comment by sanni** [comment]
Every adapter I made is here: https://github.com/sanni/cartreader/tree/master/hardware
Adapters made by the community can be found here: https://github.com/sanni/cartreader/discussions/354

Any adapter that is missing you have to either make yourself or wait until someone else designs it for you.

I have written a tutorial on how to design your own adapters here: https://github.com/sanni/cartreader/wiki/Designing-your-own-Adapters
